src/scenes/level_scene.py

```diff
@@ -1980,6 +1980,20 @@ def left_click(self, position: Position) -> None:
         """
         if self.menu_manager.active_menu:
             # TODO: check if the raw value could be replaced by a meaningful constant
+            # Check if the click is on menu
+            menu_position = self.menu_manager.active_menu.determine_position(self.active_screen_part)
+            if menu_position != None:
+                my_rect = pygame.Rect(
+                    menu_position.x, menu_position.y,
+                    self.menu_manager.active_menu._InfoBox__size[0],
+                    self.menu_manager.active_menu._InfoBox__size[1]
+                )
+                if (
+                    self.menu_manager.active_menu is not None
```

Comment:
You don't need this check.
Line 1981 is checking if the `active_menu` is defined, this whole code block is not going to be executed if `active_menu` is not defined.

Reply:
It's my fault. Thank you for your patience and care to a beginner.